pkg/controller/endpoint/endpoints_controller.go

```diff
@@ -60,6 +60,11 @@ const (
 	// 5ms, 10ms, 20ms, 40ms, 80ms, 160ms, 320ms, 640ms, 1.3s, 2.6s, 5.1s, 10.2s, 20.4s, 41s, 82s
 	maxRetries = 15
 
+	// maxCapacity represents the maximum number of addresses that should be
+	// stored in an Endpoints resource. In a future release, this controller
```

Comment:
Endpoints ->endpoints

Reply:
Thanks for the suggestion! This was actually intentional. I try to capitalize resource names to help differentiate them from other concepts, especially when it's a generic word like "endpoints". I've seen this same approach used throughout Kubernetes and actually picked it up from docs reviews, but it is admittedly a bit inconsistent. In this file, resource names look like they're capitalized around half of the time. If there's any formal guidance on this, it would be great to be more consistent one way or the other.